Fix Z-normalization on 'y' (labels)

peterLingZijie opened the pull request:
Why:

1. Add z-normalization on `y_train` and `y_test`, as implemented in He Tong's MM
2. Add explanation in Evaluation part why z-normalization is not reversed on `y_train` and `y_test`
3. Test the code with later Python packages, including Numpy, Scipy, PyTorch and Scikit-learn

This change addresses the need by:

CBIG and external users

peterLingZijie committed:
Fix Z-normalization on 'y' (labels)

Why:

1. Add z-normalization on `y_train` and `y_test`, as implemented in He Tong's MM
2. Add explanation in Evaluation part why z-normalization is not reversed on `y_train` and `y_test`
3. Test the code with later Python packages, including Numpy, Scipy, PyTorch and Scikit-learn

This change addresses the need by:

CBIG and external users

anlijuncn commented on rs-fMRI/v1.0/meta_matching_v1.0.ipynb:
```diff
@@ -141,6 +141,40 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Make it more specific. "After data split, we z-normalize the label (y) of training and test sets with training mean and std."

anlijuncn commented on rs-fMRI/v1.1/meta_matching_v1.1.ipynb:
```diff
@@ -148,6 +148,40 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Same, let the user know which variable you normalized, using which stats (mean and std).

anlijuncn commented on rs-fMRI/v2.0/meta_matching_v2.0.ipynb:
```diff
@@ -133,6 +133,40 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Same, let the user know which variable you normalized, using which stats (mean and std).

anlijuncn reviewed (changes requested):
Make y normalization part more straightforward.

peterLingZijie committed:
Fix Z-normalization on 'y' (labels)

Why:

Make the z-norm description more specific

This change addresses the need by:

CBIG and external users
